core/src/main/resources/license-deprecated.json

```diff
@@ -0,0 +1,13 @@
+{
+    "GPL-2.0": "GPL-2.0-only",
+    "GPL-2.0+": "GPL-2.0-or-later",
+    "GPL-2.0-with-classpath-exception": "GPL-2.0-only WITH Classpath-exception-2.0",
```

Comment:
Is the final -2.0 required, or can it be "GPL-2.0-only WITH Classpath-exception" ? Not for any particular reason, I find it repetitive to have it repeated 2.0.

Reply:
Ok thanks for the explanation!